src/main/java/com/evolveum/polygon/connector/msgraphapi/UserProcessing.java

```diff
@@ -1270,32 +1373,97 @@ public String getSelectorSingle(OperationOptions options) {
+                    ATTR_ACCOUNTENABLED,
+                    ATTR_DISPLAYNAME,
+                    ATTR_ONPREMISESIMMUTABLEID,
+                    ATTR_MAILNICKNAME,
+                    ATTR_USERPRINCIPALNAME,
+                    ATTR_ABOUTME,
+                    ATTR_BIRTHDAY,
+                    ATTR_CITY,
+                    ATTR_COMPANYNAME,
+                    ATTR_COUNTRY,
+                    ATTR_DEPARTMENT,
+                    ATTR_GIVENNAME,
+                    ATTR_HIREDATE,
+                    ATTR_IMADDRESSES,
+                    ATTR_ID,
+                    ATTR_INTERESTS,
+                    ATTR_JOBTITLE,
+                    ATTR_MAIL,
+                    ATTR_MOBILEPHONE,
+                    ATTR_MYSITE,
+                    ATTR_OFFICELOCATION,
+                    ATTR_ONPREMISESLASTSYNCDATETIME,
+                    ATTR_ONPREMISESSECURITYIDENTIFIER,
+                    ATTR_ONPREMISESSYNCENABLED,
+                    ATTR_PASSWORDPOLICIES,
+                    ATTR_PASTPROJECTS,
+                    ATTR_POSTALCODE,
+                    ATTR_PREFERREDLANGUAGE,
+                    ATTR_PREFERREDNAME,
+                    ATTR_PROXYADDRESSES,
+                    ATTR_RESPONSIBILITIES,
+                    ATTR_SCHOOLS,
+                    ATTR_SKILLS,
+                    ATTR_STATE,
+                    ATTR_STREETADDRESS,
+                    ATTR_SURNAME,
+                    ATTR_USAGELOCATION,
+                    ATTR_USERTYPE,
+                    ATTR_ASSIGNEDLICENSES,
+                    ATTR_EXTERNALUSERSTATE,
+                    ATTR_EXTERNALUSERSTATECHANGEDATETIME,
+                    ATTR_MANAGER,
+                    ATTR_EMPLOYEE_HIRE_DATE,
+                    ATTR_EMPLOYEE_LEAVE_DATE_TIME,
+                    ATTR_EMPLOYEE_TYPE,
+                    ATTR_FAX_NUMBER,
+                    ATTR_EMPLOYEE_ID,
+                    ATTR_ONPREMISESEXTENSIONATTRIBUTES
+            ));
         } else {
 
-            return selector(ATTR_ACCOUNTENABLED, ATTR_DISPLAYNAME,
-                    ATTR_ONPREMISESIMMUTABLEID, ATTR_MAILNICKNAME, ATTR_USERPRINCIPALNAME,
-                    ATTR_CITY, ATTR_COMPANYNAME, ATTR_COUNTRY, ATTR_DEPARTMENT,
-                    ATTR_GIVENNAME, ATTR_IMADDRESSES, ATTR_ID,
-                    ATTR_JOBTITLE, ATTR_MAIL, ATTR_MOBILEPHONE, ATTR_OFFICELOCATION,
-                    ATTR_ONPREMISESLASTSYNCDATETIME, ATTR_ONPREMISESSECURITYIDENTIFIER,
-                    ATTR_ONPREMISESSYNCENABLED, ATTR_PASSWORDPOLICIES,
-                    ATTR_POSTALCODE, ATTR_PREFERREDLANGUAGE,
-                    ATTR_PROXYADDRESSES, ATTR_STATE, ATTR_STREETADDRESS, ATTR_SURNAME,
-                    ATTR_USAGELOCATION, ATTR_USERTYPE, ATTR_ASSIGNEDLICENSES,
-                    ATTR_EXTERNALUSERSTATE, ATTR_EXTERNALUSERSTATECHANGEDATETIME, ATTR_MANAGER);
-
+            return selector(
```

Comment:
please avoid linebreak and formatting changes

Reply:
done